### PR TITLE
Newsletter subscriptions count should be configurable

### PR DESCRIPTION
**Pull request by therouv**
If you'll send a newsletter via Magento core functionality, Magento sends 20 emails per default and waits for the next cronjob to send the next 20 mails and so on. It would be great to make the two variables `$countOfQueue` and `$countOfSubscritions` configurable via system configuration.

### CODE REVIEW

File: app/code/core/Mage/Newsletter/etc/system.xml [hunk 1]
```diff
@@ -118,6 +118,32 @@
                         </un_email_template>
                     </fields>
                 </subscription>
+                <sending translate="label">
+                    <label>Sending Options</label>
+                    <frontend_type>text</frontend_type>
+                    <sort_order>10</sort_order>
+                    <show_in_default>1</show_in_default>
+                    <show_in_website>1</show_in_website>
```
Comment (LeeSaferite) [inline app/code/core/Mage/Newsletter/etc/system.xml:126]: Delete this line

File: app/code/core/Mage/Newsletter/etc/system.xml [hunk 2]
```diff
@@ -118,6 +118,32 @@
...
+                    <show_in_store>1</show_in_store>
```
Comment (LeeSaferite) [inline app/code/core/Mage/Newsletter/etc/system.xml:127]: Delete this line

File: app/code/core/Mage/Newsletter/etc/system.xml [hunk 3]
```diff
@@ -118,6 +118,32 @@
...
+                    <fields>
+                        <count_of_queue translate="label">
+                            <label>Count of Queue per Sending Process</label>
+                            <frontend_type>text</frontend_type>
+                            <sort_order>1</sort_order>
+                            <show_in_default>1</show_in_default>
+                            <show_in_website>1</show_in_website>
```
Comment (LeeSaferite) [inline app/code/core/Mage/Newsletter/etc/system.xml:134]: Delete this line

File: app/code/core/Mage/Newsletter/etc/system.xml [hunk 4]
```diff
@@ -118,6 +118,32 @@
...
+                            <show_in_store>1</show_in_store>
```
Comment (LeeSaferite) [inline app/code/core/Mage/Newsletter/etc/system.xml:135]: Delete this line

File: app/code/core/Mage/Newsletter/etc/system.xml [hunk 5]
```diff
@@ -118,6 +118,32 @@
...
+                        </count_of_queue>
+                        <count_of_subscriptions translate="label">
+                            <label>Count of Queue per Sending Process</label>
+                            <frontend_type>text</frontend_type>
+                            <sort_order>2</sort_order>
+                            <show_in_default>1</show_in_default>
+                            <show_in_website>1</show_in_website>
```
Comment (LeeSaferite) [inline app/code/core/Mage/Newsletter/etc/system.xml:142]: Delete this line

File: app/code/core/Mage/Newsletter/etc/system.xml [hunk 6]
```diff
@@ -118,6 +118,32 @@
...
+                            <show_in_store>1</show_in_store>
```
Comment (LeeSaferite) [inline app/code/core/Mage/Newsletter/etc/system.xml:143]: Delete this line

**Comment by LeeSaferite** [comment]
There is no reason to have these values be configurable at the website and/or store level.  This is because the cron job does not run once as each store.

**Comment by therouv** [comment]
True. I'll change that and send a new pull request.

**Comment by therouv** [comment]
Please see new pull request #99.

**Pull request closed without merging**